src/components/map/layers/NetworkLayers.tsx

```diff
@@ -100,7 +101,16 @@ export default class NetworkLayers extends Component<INetworkLayersProps> {
 
         return {
             // Layer name 'solmu' is directly mirrored from Jore through geoserver
-            solmu: (properties: any, zoom: any) => {
+            solmu: (properties: any, zoom: number) => {
+
+                const selectedTransitTypes = toJS(this.props.networkStore!.selectedTransitTypes);
+                if (properties.transittypes) {
+                    const transitTypes = TransitTypeHelper
```

Comment:
We already have "selectedTransitTypes so this should be names as "propertyTransitTypes" or something

Reply:
Good point, renaming